### PR TITLE
chore: generate remoteDetectionAlert schema, chore: update @comapeo/schema to v1.1.1

### PR DESCRIPTION
**Pull request by tomasciccola**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mapeo/mock-data",
-  "version": "1.0.3-alpha.7",
+  "version": "2.0.0",
```
Comment (EvanHahn) [inline package.json:3]: Should this instead be done in the release script?

**Review by EvanHahn — CHANGES_REQUESTED** [review]
I don't think this works with `remoteDetectionAlert`s. This script doesn't work for me:

```javascript
import { generate } from './index.js'

console.log(generate('remoteDetectionAlert')[0])
```

I think we need to do a little extra to make this happen.

**Comment by EvanHahn** [comment]
I think this is not ready for review, but let me know if that's wrong.

File: package.json (hunk 1 above)
Comment (tomasciccola) [reply package.json]: I dunno know how this change slipped in there. Fixed on [700b27eca3ca348edf57b1895a3fafc696d2a3a9](https://github.com/digidem/mapeo-mock-data/pull/21/commits/700b27eca3ca348edf57b1895a3fafc696d2a3a9)

**Comment by tomasciccola** [comment]
> I think this is not ready for review, but let me know if that's wrong.

I think this may now be ready for review? 
We can now generate `RemoteDetectionAlert` docs. Its kind of weird how I solved the `$ref` to Geometry. I basically inline the schema by loading it manually and then resolve its definitions with `dereference-json-schema`. I still get a type error because we're deleting the `$ref` field which is a readonly string const. So if you have any ideas on how to avoid that (now there's a tsignore in there)...

**Comment by EvanHahn** [comment]
Fixed in 03eb40c380fdf1af9bbd77f1d29859e8b89266d8. Merging!

**Review by EvanHahn — APPROVED** [review]

**Comment by tomasciccola** [comment]
> Fixed in [03eb40c](https://github.com/digidem/mapeo-mock-data/commit/03eb40c380fdf1af9bbd77f1d29859e8b89266d8). Good to merge!

ha! had the same non-fix ('cause I forgot to add `$ref: Geometry.$id`)

**Pull request merged**